packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -591,25 +591,25 @@ $Calendar-dayPicker-margin: 10px;
 }
 
 .monthIsHighlighted {
-  background-color: $ms-color-themeLight;
+  background-color: $ms-color-themeLight !important;
```

Comment:
!important raises red flags - should be a last resort

Reply:
agreed. this will break anyone downstream that is trying to theme their calendar.